test/rest-adapter.test.js

```diff
@@ -460,6 +460,23 @@ describe('RestAdapter', function() {
 
     });
 
+    it('should find RestMethod by name', function(done) {
+      var name = 'testClass.testMethod';
+      var _res;
```

Comment:
I see res is already defined at the top invoke() parent test. That is why all the children are using _res.